data/2014.schedule_data.rb

```diff
@@ -0,0 +1,31 @@
+# <img src="images/Greenville.jpeg" alt="Greenville PirateFest 2014">
+# <h2>2014 Performance Schedule</h2>
+Gig.create!(published: true, date: "Mar 1, 2014", name: "[Garner Talent Showcase](http://www.garnerperformingartscenter.com/EventInformation.asp)", location:  "Garner Performing Arts Center")
```

Comment:
I'd probably tend to put the structured data in a CSV file with an importer rake task to import and set up the models.